packages/core/admin/admin/src/content-manager/components/BlocksInput/Blocks/List.tsx

```diff
@@ -29,13 +29,13 @@ const listStyle = css`
   }
 `;
 
-const Orderedlist = styled.ol`
-  list-style-type: decimal;
+const Orderedlist = styled.ol<{ listStyle: CSSProperties['listStyleType'] }>`
+  list-style-type: ${(props) => props.listStyle};
   ${listStyle}
 `;
```

Comment:
it's slightly confusing to have the same var/prop name for the common styles and the list style type